internal/olm/manager.go

```diff
@@ -26,16 +26,17 @@ import (
 )
 
 const (
-	DefaultVersion = "latest"
-	DefaultTimeout = time.Minute * 2
+	DefaultVersion   = "latest"
+	DefaultTimeout   = time.Minute * 2
+	DefaultNamespace = "olm"
```

Comment:
```suggestion
	DefaultOlmNamespace = "olm"
```

Reply:
I haven't changed this because, in all places its called as `olm.DefaultNamespace` which reads better than `olm.DefaultOLMNamespace`. Instead I have added a comment to struct to be explicit about its purpose

Reply:
if this doesn't look good, I can change it

Reply:
Could you please just add in the comment tthe info over the operator files (OLM operator bundle) will be applied there as well?

Reply:
The place call olm, however, we have Operator Namespace, Watch Namespaces, and then here the Namespace where OLM will be installed. So, I really think that is required we make clear that it is the OLM Namespace since it could lead us to so many possibilities.

Reply:
I think I have updated all the relevant places to OLMNamespace and updated the comments. Also I think I've addressed all the review comments and rebased. If you think I missed something can you pls point me to that.

Reply:
All fine. Really tks.